### DISCUSSION TITLE
trigger AuthenticationStatusChanged when success login?

### DISCUSSION THREAD

**Discussion by hanafiahtkj** [aliyazdi75/gallery#3] [Q&A] [answered]
where to put trigger event "AuthenticationStatusChanged" when login success?

**Comment by aliyazdi75** [comment, marked as answer]
Sorry in this project authentication on the server-side has to been implemented and it's not completed here eighter, you can check my other project.
- Here https://github.com/aliyazdi75/flutter_chat/blob/main/lib/presentation/screens/splash/splash.dart#L45 you can see the `AuthenticationStatusCheckRequested` checks whenever the app opens in the splash.
- Here https://github.com/aliyazdi75/flutter_chat/blob/main/lib/blocs/socket/bloc.dart#L51 triggers `UnAuthorizedRequested` whenever our token has been deleted or expired in each `bloc` we want to log the user out of the system and the authentication state changes in splash and user navigated there when that happens.

**Comment by hanafiahtkj** [comment]
You are really awesome bro. You save my live...

**Comment by aliyazdi75** [comment]
Thanks, Happy to hear that :blue_heart:
